services/brig/schema/src/V47.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes       #-}
+
+module V47 (migration) where
+
+import Cassandra.Schema
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 47 "Create and populate prefix table" $
```

Comment:
This migration doesn't actually populate the table, we need to perform a manual step here.